fearless/Modules/Staking/ValidatorInfo/View/ValidatorInfoEmptyStakeCell.swift

```diff
@@ -0,0 +1,50 @@
+import UIKit
+import SoraUI
+
+final class ValidatorInfoEmptyStakeCell: UITableViewCell {
+    private enum Constants {
+        static let verticalInset: CGFloat = 0
```

Comment:
zero inset is just no inset😏
![image](https://user-images.githubusercontent.com/17689921/116688431-5e3f8180-a9bf-11eb-946a-57eaf7de485a.png)

Reply:
It kinda works 
![image](https://user-images.githubusercontent.com/3176149/116691583-ab255700-a9c3-11eb-847e-cdd97662dab7.png)